source/docs/zero-to-robot/step-2/wpilib-setup.rst

```diff
@@ -1,280 +1,108 @@
 WPILib Installation Guide
-=============================
+=========================
 
-.. note:: This setup is for C++/Java teams. LabVIEW teams should proceed to :doc:`Installing LabVIEW for FRC </docs/zero-to-robot/step-2/labview-setup>`.
+This guide is intended for Java and C++ teams. LabVIEW teams can skip to :doc:`labview-setup`. Additionally, the below tutorial shows Windows 10, but the steps are identical for all operating systems. Notes differentating operating systems will be shown.
 
-.. tabs::
-
-  .. tab:: Windows
-    .. note:: Windows 7: You **must** install the NI FRC Game Tools or .NET Version 4.62 (or later) before proceeding with the install of Visual Studio Code for FRC. The NI FRC Game Tools installer will automatically install the proper version of .NET. The standalone .NET installer is `here <https://support.microsoft.com/en-us/help/3151800/the-net-framework-4-6-2-offline-installer-for-windows>`__
```

Comment:
I'm guessing this is still needed, unless Avalonia needs an even later version of .NET

Reply:
Resolved in 9140e1c